plugins/legacy-preset-chart-nvd3/src/utils.js

```diff
@@ -132,8 +132,40 @@ export function generateRichLineTooltipContent(d, timeFormatter, valueFormatter)
   return dompurify.sanitize(tooltip);
 }
 
-export function generateAreaChartTooltipContent(d, timeFormatter, valueFormatter) {
-  const total = d.series[d.series.length - 1].value;
+export function generateCompareTooltipContent(d, valueFormatter) {
+  let tooltip = '';
+  tooltip +=
+    "<table><thead><tr><td colspan='3'>" +
+    `<strong class='x-value'>${d.value}</strong>` +
```

Comment:
Template literals support multiline strings directly so the manual concatenation might not be needed.

Reply:
The other tooltip generator use this style so I am gonna leave it same for now. 
Don't want to change all of them yet. 
Also to avoid potential unintentional spacing from space characters between tags.